src/SignalRServiceExtension/Client/AzureSignalRClient.cs

```diff
@@ -197,10 +197,10 @@ private static IEnumerable<Claim> BuildJwtClaims(IEnumerable<Claim> customerClai
         private async Task InvokeAsync(ServiceEndpoint[] endpoints, Func<IInternalServiceHubContext, Task> func)
         {
             var serviceHubContext = await GetHubContextAsync();
```

Comment:
Directly make it IInternalServiceHubContext, instead of do it in next check?

Reply:
`IInternalServiceHubContext` doesn't implement `IServiceHubContext` or `ServiceHubContext` abstract class..... I am thinking about whether making `IIternalServiceHubContext` an extension method or an class implementing `ServiceHubContextImpl`.